pkg/ibmvpc/vpc.go

```diff
@@ -256,6 +252,16 @@ func (nl *NaclLayer) RulesInConnectivity(src, dst vpcmodel.Node,
 	return allowRes, denyRes, nil
 }
 
+// FilterKindName of NaclLayer used in printing, e.g. explainability
+// differs from Kind() which returns the const string NaclLayer used by the code
+func (nl *NaclLayer) FilterKindName() string {
+	return "network ACL"
```

Comment:
why do we need to "duplicate" strings "NACL" and "network ACL" , instead of using the same const both for `Kind()` of `NACL` and for `FilterKindName` of `NaclLayer`?

Reply:
Two reasons:
1. One is in the level of the layer and the other is in the level of the table (each layer has many tables). What I need here is the layer. If will be very messy and unnatural to use for the header a string from a specific table (e.g. specific nacl)
2. While Kind() is used by the code, the string here is for printing and we may want to use a different string.

Reply:
> One is in the level of the layer and the other is in the level of the table

But in the level of the layer this method returns the kind of a specific table's level , which is expected to be consistent with Kind() for a specific table 

>  the string here is for printing

If this method is only for printing in explainability, then maybe it should not be defined here, but rather as a function in `explainabilityPrint.go`, that does a switch over `FilterTrafficResource.Kind()`, and returns the relevant string to print?

Reply:
> > One is in the level of the layer and the other is in the level of the table
> 
> But in the level of the layer this method returns the kind of a specific table's level , which is expected to be consistent with Kind() for a specific table
Printing the string returned by Kind() is ugly

> 
> > the string here is for printing
> 
> If this method is only for printing in explainability, then maybe it should not be defined here, but rather as a function in `explainabilityPrint.go`, that does a switch over `FilterTrafficResource.Kind()`, and returns the relevant string to print?

The method is also used by other member functions of FilterTrafficResource so it can't be defined in explainability 

Anyways, given the time and text we have already spent over this issue - I've committed a version in which I use Kind() instead. I think the resulted output is ugly, but see for yourself. If you like it we'll keep it.

Reply:
> The method is also used by member functions of FilterTrafficResource so it can't be defined in explainability

why ? pkg ibmvpc already imports and uses functions from `vpcmodel`

Reply:
ok, moved to explainability